frontend/src/app/components/resizer/wp-resizer.component.ts

```diff
@@ -26,29 +26,36 @@
 // See docs/COPYRIGHT.rdoc for more details.
 //++
 
-import {Component, ElementRef, HostListener, Input, OnInit} from '@angular/core';
-import {distinctUntilChanged} from 'rxjs/operators';
+import {AfterViewInit, Component, ElementRef, Input, OnInit} from '@angular/core';
+import {debounceTime, distinctUntilChanged} from 'rxjs/operators';
 import {TransitionService} from '@uirouter/core';
 import {MainMenuToggleService} from "core-components/main-menu/main-menu-toggle.service";
 import {BrowserDetector} from "core-app/modules/common/browser/browser-detector.service";
 import {UntilDestroyedMixin} from "core-app/helpers/angular/until-destroyed.mixin";
+import {ResizeDelta} from "core-app/modules/common/resizer/resizer.component";
+import {fromEvent} from "rxjs";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/d356c82449c2301a333e72b8f2c7e05463706d4a/frontend/src/app/components/resizer/wp-resizer.component.ts#L37"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>